### PR TITLE
Add documentation for is_feasible()

### LINKED ISSUE #208: Documentation for feasibility check
Provide some high-level documentation for how/when to use the feature added in #203 . This could be added to an existing page in the docs. The bulk of the text could be in the code itself with a Sphinx 'autodoc' directive to expose it in the docs pages.

### PR DESCRIPTION
**Pull request by tarnold17**
## Fixes/Addresses/Summary/Motivation:

Add documentation for `is_feasible()` function. Resolves #208.

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my
contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.md file
   at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has
   rights to intellectual property that includes these contributions, I represent that I have
   received permission to make contributions and grant the required license on behalf of that
   employer.

### CODE REVIEW

File: docs/utilities/Results.rst [hunk 1]
```diff
@@ -298,4 +300,37 @@ Example of how this method can be used::
...
+Is Feasible
+------------
+
+**Method Description**
+
+Verifies the solution contained in a Pyomo model object is feasible. This requires iterating through all variables and constraints and ensuring that the constraint and variable bounds are satisfied at the solution present in the model.
+
+**How to Use**
+
+The method requres that a Pyomo model be passed as the first argument. Two other keyword arguments are optional:
```
Comment (melody-shellman) [inline docs/utilities/Results.rst:317]:
```suggestion
The method requires that a Pyomo model be passed as the first argument. Two other keyword arguments are optional:
```

File: docs/utilities/Results.rst [hunk 2]
```diff
@@ -298,4 +300,37 @@ Example of how this method can be used::
...
+
+2. *"cons_tol"* - Violation tolerance for constraints. Default value is 1e-3.
+
+
+Example of how this method can be used::
+
+  from pareto.utilities.results import is_feasible, nostdout
+
+  # Assume model is any Pyomo model that has already been created and solved
+
```
Comment (melody-shellman) [inline docs/utilities/Results.rst:329]: The "nostdout()" is a function that is defined in our code. While maybe self-explanatory, I suggest adding a comment
```suggestion
# With statement suppresses messages from the method call. 
```

**Review by melody-shellman — COMMENTED** [review]
Typo and small suggestion!

File: docs/utilities/Results.rst (hunk 2 above)
Comment (tarnold17) [reply docs/utilities/Results.rst:329]: Good suggestion - addressed!

**Review by melody-shellman — APPROVED** [review]
Looks good!

**Pull request merged**